rpi-update

```diff
@@ -26,6 +26,16 @@ fi
 
 BRANCH=${BRANCH:-"master"}
 ROOT_PATH=${ROOT_PATH:-"/"}
+CUR_FW_SUBDIR="/$(vcgencmd get_config os_prefix | cut -d'=' -f2)"
```

Comment:
I think old firmware (probably including stable) will produce "os_prefix is unknown" here

Reply:
Good point - I'll add an explicit check.